paperqa/contrib/openreview_paper_helper.py

```diff
@@ -0,0 +1,165 @@
+import json
+import logging
+import os
+from pathlib import Path
+from typing import Any
+
+import httpx
+from litellm import completion
```

Comment:
Okay so `litellm` isn't actually a direct dependency of `paper-qa`, we use [`lmi`](https://github.com/Future-House/ldp/tree/v0.25.1/packages/lmi) (on PyPI as `fhlmi`).
 
Would you be open to trying to use `lmi`'s `lmi.LiteLLMModel`'s `call_single` over `litellm`'s `completion`?  Otherwise we can just make the change after merging this PR